masterlist.yaml

```diff
@@ -226,7 +226,7 @@ common: []
 bash_tags: []
 
 globals:
-# Latest LOOT Thread
+  # Latest LOOT Thread
```

Comment:
Not related to adding merged objects, so it should be done in a separate pull request,
```suggestion
# Latest LOOT Thread
```